### PR TITLE
Feat/storybook_fixes

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

merging into `upgrades` feature branch

majority of changes fixing the import paths
does not have proptable or story src plugins working
these are minimally used and its actually easier to just look at the code right now
(props are displayed poorly, story src is a single level of display - no insight into internals)

- fixes imports for storybook
- upgraded storybook

**Type of Change**

- [ ] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- UI Kit (select one):
  - [ ] New Components with storybook stories created
  - [ ] Modified Existing components and updates stories
  - [ ] No new UIKit components or changes
- [ ] Feature is minimally responsive
- [ ] Manual testing of UI feature
- [ ] Add copyrights to new files
- [ ] Connected ticket to PR

### CODE REVIEW

File: src/Button/GoogleLogin/stories.tsx [hunk 1]
```diff
@@ -18,11 +18,7 @@
  */
 
 import { storiesOf } from '@storybook/react';
+import { GoogleLogin } from './index';
```
Comment (ciaranschutte) [inline src/Button/GoogleLogin/stories.tsx:21]: couldn't pin down why, but some of the story files need the import with `./index` explicitly defined, others are fine as `.`

**Comment by ciaranschutte** [comment]
build is breaking due to lack of typings. i will tweak the CI to allow for now. we can add typings as we see fit.

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// export const parameters = {
```
Comment (demariadaniel) [inline .storybook/preview.js:1]: Remove commented code? Or is this related to the upcoming plugin work?

Comment (ciaranschutte) [reply .storybook/preview.js]: nope - goood catch ty - updated @demariadaniel

File: src/Icon/index.tsx [hunk 3]
```diff
@@ -38,6 +38,7 @@ export const Icon: React.ComponentType<
   } & SVGAttributes<SVGElement>
 > = ({ name, width, height, fill, className, title, outline, ...rest }) => {
   const theme = useTheme();
+  console.log('naaaaaaaame', name);
```
Comment (demariadaniel) [inline src/Icon/index.tsx:41]: Remove console statement?

File: src/Pipe/index.tsx [hunk 4]
```diff
@@ -60,3 +60,5 @@ const Pipe: React.ComponentType<{}> & { Item: typeof PipeItem } = ({ children })
 );
 
 Pipe.Item = PipeItem;
+
+export { Pipe };
```
Comment (demariadaniel) [inline src/Pipe/index.tsx:64]: This breaks when testing on localhost
`TypeError: Cannot read properties of undefined (reading 'Item')`

File: src/VerticalTabs/stories.tsx [hunk 5]
```diff
@@ -38,7 +37,7 @@ const LoremTooltip = (
   </>
 );
 
-const VerticalTabsStories = storiesOf(`${__dirname}`, module).add('Basic', () => {
+storiesOf(`VerticalTabs`, module).add('Basic', () => {
   const [activeItem, setActiveItem] = React.useState(0);
```
Comment (demariadaniel) [inline src/VerticalTabs/stories.tsx:41]: Not sure which line but Vertical Tabs break on localhost
`TypeError: Cannot read properties of undefined (reading 'Item')`

File: src/utils/Hook/stories.tsx [hunk 6]
```diff
@@ -17,15 +17,14 @@
...
-import Button from '../../Button';
 import Hook from '.';
+import { Button } from '../../Button';
 import readme from './readme.md';
 
-const HookStories = storiesOf(`${__dirname}`, module).add(
+storiesOf(`Hook`, module).add(
   'Basic',
   () => (
     <Hook
```
Comment (demariadaniel) [inline src/utils/Hook/stories.tsx:30]: Failing w:
`Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

File: src/utils/useClickAway/stories.tsx [hunk 7]
```diff
@@ -19,10 +19,10 @@
 
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import useClickAway from '.';
+import { useClickAway } from '.';
 import { action } from '@storybook/addon-actions';
 
-export default storiesOf(`${__dirname}`, module).add(
+storiesOf(`useClickAway`, module).add(
```
Comment (demariadaniel) [inline src/utils/useClickAway/stories.tsx:25]: Error: `TypeError: Object(...) is not a function`

Comment (ciaranschutte) [reply src/utils/useClickAway/stories.tsx:25]: import './' vs import './index' fixed

File: src/utils/Hook/stories.tsx (hunk 6 above)
Comment (ciaranschutte) [reply src/utils/Hook/stories.tsx:30]: import './' vs import './index' fixed

File: src/VerticalTabs/stories.tsx (hunk 5 above)
Comment (ciaranschutte) [reply src/VerticalTabs/stories.tsx:41]: could not recreate this one

**Comment by ciaranschutte** [comment]
@demariadaniel tackled anything storybook related I could re-create
there are a handful of odd controls that don't quite work but it's in sync with what we currently have in our live version
i'll have another PR to fix typescript issues / setup CI

File: src/VerticalTabs/stories.tsx (hunk 5 above)
Comment (demariadaniel) [reply src/VerticalTabs/stories.tsx:41]: Will resolve this one offline (also occurring in Pipe) -- storybook is building and running so this is specific to the components

**Review by demariadaniel — APPROVED** [review]

**Comment by Buwujiu** [comment]
Some issues need to be fixed, but storybook is working. Will merge soon.
